runtime/lua/vim/lsp/util.lua

```diff
@@ -26,7 +26,7 @@ local default_border = {
 --- size it adds to a float.
 ---@param opts table optional options for the floating window
 ---            - border (string or table) the border
----@return table size of border in the form of { height = height, width = width }
+---@return { height: integer, width: integer }
```

Comment:
This is a good catch though, should still get this in (maybe in https://github.com/neovim/neovim/pull/25171)